data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,276 @@
+
+/* Calculate the size of the largest table, in MiB. Replace ${schema_name} with your schema name. */
+SELECT table_name,table_schema,SUM(data_length)/1024/1024 AS data_length,SUM(index_length)/1024/1024 AS index_length,SUM(data_length+index_length)/1024/1024 AS SUM from information_schema.tables WHERE table_schema = "${schema_name}" GROUP BY table_name,table_schema ORDER BY SUM DESC LIMIT 5;
+```
+
+### Disk space for the target TiKV cluster
+
+The target TiKV cluster must have enough disk space to store the imported data. In addition to [the standard hardware requirements](/hardware-and-software-requirements.md), the storage space of the target TiKV cluster must be larger than **the size of the data source x [the number of replicas](/faq/deploy-and-maintain-faq.md#is-the-number-of-replicas-in-each-region-configurable-if-yes-how-to-configure-it) x 2**. For example, if the cluster uses 3 replicas by default, the target TiKV cluster must have a storage space larger than 6 times the size of the data source. The formula has `x 2` because:
+
+- Index might take extra space.
+- RocksDB has a space amplification effect.
+
+## Step 1. Export all data from MySQL
+
+1. Export all data from MySQL by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db1 -f 'my_db1.table[12]' -o ${data-dir}
+    ```
+
+    Dumpling exports data in SQL files by default. You can specify a different file format by adding the `--filetype` option.
+
+    The parameters used above are as follows. For more Dumpling parameters, refer to [Dumpling Overview](/dumpling-overview.md).
+
+    |parameters             |Description|
+    |-                      |-|
+    |`-u` or `--user`       |MySQL user|
+    |`-p` or `--password`   |MySQL user password|
+    |`-P` or `--port`       |MySQL port|
+    |`-h` or `--host`       |MySQL IP address|
+    |`-t` or `--thread`     |The number of threads used for export|
+    |`-o` or `--output`     |The directory that stores the exported file. Supports a local path or an [external storage URL](/br/backup-and-restore-storages.md)|
+    |`-r` or `--row`        |The maximum number of rows in a single file|
+    |`-F`                   |The maximum size of a single file, in MiB|
+    |-`B` or `--database`   |Specifies a database to be exported|
+    |`-f` or `--filter`     |Exports tables that match the pattern. The syntax is the same as [table-filter](/table-filter.md). For example, `[\*.\*,!/^(mysql&#124;sys&#124;INFORMATION_SCHEMA&#124;PERFORMANCE_SCHEMA&#124;METRICS_SCHEMA&#124;INSPECTION_SCHEMA)$/.\*]` exports all tables except the system schema. |
+
+    Make sure `${data-path}` has enough space to store the exported data. To prevent the import from being interrupted by a large table consuming all the spaces, it is strongly recommended to use the `-F` option to limit the size of a single file.
```

Comment:
```suggestion
    Make sure `${data-path}` has enough space to store the exported data. To prevent the export from being interrupted by a large table consuming all the spaces, it is strongly recommended to use the `-F` option to limit the size of a single file.
```